src/main/java/org/jabref/gui/maintable/MainTableColumnModel.java

```diff
@@ -67,20 +67,29 @@ public static Type fromString(String text) {
         }
     }
 
-    private final ObjectProperty<Type> typeProperty = new SimpleObjectProperty<>();
-    private final StringProperty qualifierProperty = new SimpleStringProperty("");
-    private final DoubleProperty widthProperty = new SimpleDoubleProperty(ColumnPreferences.DEFAULT_WIDTH);
+    private final ObjectProperty<Type> typeProperty;
+    private final StringProperty qualifierProperty;
+    private final DoubleProperty widthProperty;
 
     /**
      * This is used by the preferences dialog, to initialize available columns the user can add to the table.
      *
      * @param type the {@code MainTableColumnModel.Type} of the column, e.g. "NORMALFIELD" or "GROUPS"
      * @param qualifier the stored qualifier of the column, e.g. "author/editor"
      */
-    public MainTableColumnModel(Type type, String qualifier) {
+    public MainTableColumnModel(Type type, String qualifier, double width) {
         Objects.requireNonNull(type);
-        this.typeProperty.setValue(type);
-        this.qualifierProperty.setValue(qualifier);
+        typeProperty = new SimpleObjectProperty<>(type);
+        qualifierProperty = new SimpleStringProperty(qualifier);
+        widthProperty = new SimpleDoubleProperty(width);
+    }
+
+    public MainTableColumnModel(Type type, String qualifier) {
```

Comment:
Can you copy the old JavaDoc to here?

Reply:
I don't see any value in copying the java doc, especially since this is a stupid constructor that does the obvious thing ;-)